utils/Constants.ts

```diff
@@ -94,3 +94,12 @@ export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: st
   mainnetFork: { symbol: "USDC", address: ETH_USDC_ADDRESS, decimals: 6 },
   rsk: { symbol: "XUSD", address: RSK_XUSD_ADDRESS, decimals: 18 }
 };
+
+export const tokenLogoUrl: { [symbol: string]: string } = {
+  BDX: "https://blindex-static-assets.s3.filebase.com/bdx-symbol-white-stroke.svg",
```

Comment:
* NIT let's save the bucket url as a variable and reuse it on all symbols instead of hardcoded it multiple times (just in case we'll change it in the future)
* I don't like the fact there is no folder for these icons. We should create a tokens-icons folder in that bucket and locate everything there. Otherwise we'll have a mess as we'll add stuff. That means also Changing it in the API server